### ISSUE TITLE
Update pagination size and add indexing

### ISSUE THREAD

**Issue by deshraj** [Cloud-CV/EvalAI#1866]
## Current Status
We have a pagination limit of 10 currently on the frontend for `My Submission` and `View All Submissions` Page. We would like to extend this limit to 100 submissions per page. Also, we should show the submission id as well as the first column

## Deliverables
- [ ] Update pagination on `My Submissions` Page to 100
- [ ] Update pagination on `View all submissions` Page to 100
- [ ] Add the column of `Submission ID` as the left most column for both `My Submission` and `View all submissions` page

**Comment by Niklex21** [comment]
Hi, can I work on that?

**Comment by yashdusing** [comment]
I'd like to work on this issue if no one has been assigned yet

**Comment by RishabhJain2018** [comment]
Hi @yashdusing Since @Niklex21 have asked first to work on this issue, hence I will give him the first priority. If he doesn't want to work on it then you are welcome to work on it.

@Niklex21 Would you like to work on the issue?

**Comment by Niklex21** [comment]
Yes, I will work on the issue.

**Comment by RishabhJain2018** [comment]
Thanks, @Niklex21 for confirming.
@yashdusing Please feel free to take up any other unassigned issue.

**Comment by gautamjajoo** [comment]
@deshraj @RishabhJain2018 Can I work on this issue?

**Comment by deshraj** [comment]
Sure, go ahead.

**Comment by gautamjajoo** [comment]
@deshraj Hey! What is `Submission ID`?

**Comment by yongzx** [comment]
Hi, is this issue resolved? I would like to take on this issue.

**Comment by RishabhJain2018** [comment]
HI @yongzx @GauravJain98 This is open. Please let us know if you would like to work on it.

**Comment by yongzx** [comment]
Yes I would like to work on it!

**Comment by RishabhJain2018** [comment]
sure! you can work on it.

**Comment by gautamjajoo** [comment]
@yongzx Hey, are you working on it?

**Comment by yongzx** [comment]
Yes I am working on it

**Comment by yongzx** [comment]
@RishabhJain2018 

I couldn't understand what goes wrong for the HTTP Get request for clicking the next button when `challenge.submissionResult.next` in `ng-click=challenge.load(challenge.submissionResult.next)` shows the correct url.

I updated `app/base/utils.py` to the following to allow pagination size of 100.
```
class StandardResultSetPagination(PageNumberPagination):
    page_size = 100
    page_size_query_param = 'page_size'
    max_page_size = 1000


def paginated_queryset(queryset, request, pagination_class=StandardResultSetPagination()):
    '''
        Return a paginated result for a queryset
    '''
    paginator = pagination_class
    # paginator.page_size = settings.REST_FRAMEWORK['PAGE_SIZE']
    result_page = paginator.paginate_queryset(queryset, request)
    return (paginator, result_page)
```

I experienced this error without changing other codes:
```
django_1  | [2019-01-29 07:37:54] INFO "GET /api/jobs/challenge/1/challenge_phase/2/submission/?page=1 HTTP/1.1" 200 68893
django_1  | [2019-01-29 07:37:58] INFO "GET /api/jobs/challenge/1/challenge_phase/2/submission/?page=2 HTTP/1.1" 200 6273
django_1  | [2019-01-29 07:37:59] INFO "GET /api/jobs/challenge/1/challenge_phase/2/submission/?page=1 HTTP/1.1" 200 68886
django_1  | [2019-01-29 07:38:00] WARNING "GET /api/jobs/challenge/1/challenge_phase/2/submission/?page=11 HTTP/1.1" 404 26
```

I believe the error comes from the backend. Where should I look into to fix the error when `vm.load(url)` should be loading the correct `url` (that comes from `vm.submissionResult.next`)

**Comment by gautamjajoo** [comment]
@yongzx Did you update in `settings/common.py`? You can try doing that.

**Comment by kurianbenoy** [comment]
Can I work on this issue @RishabhJain2018 @deshraj ?

**Comment by gautamjajoo** [comment]
@kurianbenoy Please take up any other issue @yongzxhas has already made a PR for it.

**Comment by yongzx** [comment]
@kurianbenoy yes I have implemented all the features and I am just waiting for the PR to get accepted.

**Comment by RishabhJain2018** [comment]
Closing this issue, since this is solved by #2199
